TBS Updates Inline Registration Hook response samples (#3467)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/reference/registration-hook/index.md

}
```

## Sample SSR responses

The external service responds to Okta indicating whether to accept the end user's self-registration attempt. The response returns a `commands` object in the body of the HTTPS response. This object contains specific syntax that indicates whether the user is allowed or denied to self-register.

See [response properties](#response-objects-that-you-send) for full details.

### Sample SSR response using data.userProfile

> The following sample response shows a successful self-registration using a valid email domain:

Reviewer: ```suggestion
The following sample response shows a successful self-registration:
```

Reviewer: I'd remove the "valid email domain"; that was the example we used for the guide, but it could revolve around any logic on the external server.

```json
{
    "commands": [
        {
            "type": "com.okta.user.profile.update",
            "value": {
                "login": "first.last@example.com"
            }
        }
    ]
}
```

### Sample SSR response using data.action

> The following sample response uses the `DENY` value and a custom error caused by an invalid email domain:

Reviewer: In this case it's accurate based on what you see in the error object. But, in theory, all sorts of reasons you might want to deny someone. That said, I think it's good to display the error object here as it's the way we pass back details to the UI.

}
```

## Sample progressive enrollment responses

The external service responds to Okta indicating whether to accept the end user's profile update. The response returns a `commands` object in the body of the HTTPS response. This object contains specific syntax that indicates whether the user is allowed or denied to update their profile with Okta.

See [response properties](#response-objects-that-you-send) for full details.

### Sample progressive enrollment response using data.UserProfileUpdate

> The following sample response shows an end user successfully adding their employee number to their profile:

Reviewer: Somewhat pedantic, but it's the code that's adding the employeeNumber to the profile. Maybe something like "The following sample response shows the addition of an employee number to an end user's profile."

Author: Thank you, Brian :)

> The following sample response shows an end user successfully adding their employee number to their profile:

Author: ```suggestion
The following sample response shows the addition of an employee number to an end user's profile:
```
